Content.Server/Administration/AdminSystem.cs

```diff
@@ -25,11 +27,28 @@ public override void Initialize()
             _adminManager.OnPermsChanged += OnAdminPermsChanged;
             SubscribeLocalEvent<PlayerAttachedEvent>(OnPlayerAttached);
             SubscribeLocalEvent<PlayerDetachedEvent>(OnPlayerDetached);
+            SubscribeLocalEvent<RoleAddedEvent>(OnRoleAddedEvent);
+        }
+
+        private void OnRoleAddedEvent(RoleAddedEvent ev)
+        {
+            if (ev.Role is TraitorRole && ev.Mind.Session != null)
```

Comment:
If you are trying to filter for antagonists here to avoid sending unnecessary data, check for role.Antagonist instead.
There are more antagonists than just traitors.

Reply:
roger